src/example.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+# Create a 2-D array, set every second element in
+# some rows and find max per row:
+
+x = np.arange(15, dtype=np.int64).reshape(3, 5)
+x[1:, ::2] = -99
+x
+# array([[  0,   1,   2,   3,   4],
+#        [-99,   6, -99,   8, -99],
+#        [-99,  11, -99,  13, -99]])
+
+x.max(axis=1)
```

Comment:
Same

Reply:
Nope. I just copy pasted the entire numpy example blindly.
Will remove after I found or write a better example